api/src/main/java/org/commonjava/indy/conf/InternalFeatureConfig.java

```diff
@@ -0,0 +1,47 @@
+package org.commonjava.indy.conf;
+
+
+import org.commonjava.propulsor.config.annotation.ConfigName;
+import org.commonjava.propulsor.config.annotation.SectionName;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.io.File;
+import java.io.InputStream;
+
+@ApplicationScoped
+@SectionName( InternalFeatureConfig.SECTION_NAME )
+public class InternalFeatureConfig implements IndyConfigInfo {
+
+    public final static String SECTION_NAME = "ssl_validation";
+
+    private Boolean sslValidation;
```

Comment:
This should be named `storeValidation`, since we want to turn on / off ALL validation, not just SSL validation, using this internal feature flag.